lib/auth_web/controllers/auth_controller.ex

```diff
@@ -183,78 +183,87 @@ defmodule AuthWeb.AuthController do
   form where they can define a new password for their account.
   """
   def login_register_handler(conn, params) do
-    # IO.inspect(params, label: "params:130")
     p = params["person"]
     email = p["email"]
     state = p["state"]
-    # IO.inspect(email, label: "email")
+
     # email is blank or invalid:
     if is_nil(email) or not Fields.Validate.email(email) do
-      conn # email invalid, re-render the login/register form:
-      |> index(params)
+      # email invalid, re-render the login/register form:
+      index(conn, params)
```

Comment:
makes sense. dunno why I left this redundant pipe in.
It has another function before invoking `index`.
Thanks for tidying.